forms/uds/a3/form_a3_ivp_error_checks_mc.csv

```diff
@@ -0,0 +1,677 @@
+a3-ivp-c-401,401,Error,a3,I,SIB20MEVAL,Conformity,SIB20MEVAL conforms,SIB20MEVAL must be an integer between 1 and 4,Q2t5. SIB20MEVAL (Sibling 20 method of evaluation) must be an integer between 1 and 4,If SIB20MEVAL<1 or SIB20MEVAL >4,,,Yes,No,
+a3-ivp-m-402,402,Error,a3,I,SIB20AGO,Missingness,SIB20AGO conditionally present,if SIBS is (>19 and <77) and SIB20ETPR is in (01-12) then SIB20AGO must be present,if Q2. SIBS (How many siblings?) is (>19 and <77) and Q2t3. SIB20ETPR (Sibling 20 primary dx) is in (01-12) then Q2t6. SIB20AGO (Sibling 20 age of onset of primary dx) must be present,If SIBS is (>19 and <77) and SIB20ETPR in (01-12) and SIB20AGO=blank,,,Yes,No,
+a3-ivp-m-403,403,Error,a3,I,SIB20AGO,Missingness,SIB20AGO conditionally blank,"if ((SIBS is <19 , 888 (na) or 999 (unknown)) or (SIB20ETPR is 00 (No neurological/psychological dx) or 99 (unknown)) then SIB20AGO must be blank","if ((Q2. SIBS (How many siblings?) is (<19 , 888 (na) or 999 (unknown)) or (Q2t3. SIB20ETPR (Sibling 20 primary dx) is 00 (No neurological/psychological dx) or 99 (unknown)) then Q2t6. SIB20AGO (Sibling 20 age of onset of primary dx) must be blank",If (SIBS is (<19 or =888 or =999) or (SIB20ETPR=00 or 99)) and SIB20AGO is not blank,,,Yes,No,
+a3-ivp-c-404,404,Error,a3,I,SIB20AGO,Conformity,SIB20AGO conforms,"SIB20AGO must be an integer between 0 and 110, or =888 (na) or 999 (unknown)","Q2t6. SIB20AGO (Sibling 20 age of onset of primary dx) must be an integer between 0 and 110, or =888 (na) or 999 (unknown)",If SIB20AGO<0 or (SIB20AGO >110 and notin 888 or 999),,,Yes,No,
+a3-ivp-m-405,405,Error,a3,I,KIDS,Missingness,KIDS present,KIDS cannot be blank,Q3. KIDS (How many children?) cannot be blank,If KIDS=blank,,,Yes,Yes,
+a3-ivp-c-406,406,Error,a3,I,KIDS,Conformity,KIDS conforms,KIDS must be an integer between 0 and 15,Q2. SIBS (How many children?) must be an integer between 0 and 15,If KIDS<0 or (KIDS >15),,,Yes,Yes,
+a3-ivp-m-407,407,Error,a3,I,KID1YOB,Missingness,KID1YOB conditionally present,if KIDS is (>0 and <77) then  KID1YOB must be present,if Q3. KIDS (How many children?) is (>0 and <77) then  Q3a1. KID1YOB (Child 1 birth year) must be present,If KIDS is (>0 and <77) and KID1YOB=blank,,,Yes,Yes,
+a3-ivp-m-408,408,Error,a3,I,KID1YOB,Missingness,KID1YOB conditionally blank,"if KIDS is (<1, 888 (na) or 999 (unknown) then  KID1YOB  must be blank","if Q3. KIDS (How many children?) is (<1 , 888 (na) or 999 (unknown) then  Q3a1. KID1YOB (Child 1 birth year)  must be blank",If KIDS is (<1 or =888 or =999) and KID1YOB is not blank,,,Yes,Yes,
+a3-ivp-c-409,409,Error,a3,I,KID1YOB,Conformity,KID1YOB conforms,"KIDYOB must be a year between 1875 and the current year, or =9999 (unknown)","Q3a1. KID1YOB (Child 1 birth year) must be a year between 1875 and the current year, or =9999 (unknown)",If KID1YOB<1875 or (KID1YOB > (current year and notin 9999)),,,No,Yes,

```

Comment:
Double checking whether 1875 is the correct minimum value here, since minimum participant YOB is 1850?